### ISSUE TITLE
6.3.0 is a crashfest

### ISSUE THREAD

**Issue by thenitai** [Snapchat/KeyDB#430]
We've been running 6.2.2 for quite some time now either as active-replica or with/without active-master.

However, 6.3 is quite a challenge as everything crashes every minute or so. Worse it crashes on each server, thus bringing down the whole set, defeating the purpose of a fail-over setup :(

There are countless reports about crashes already, but our solution was to downgrade to 6.2.2.1.

**Comment by JohnSully** [comment]
Hi @thenitai I'm sorry for your experience, there were some clear misses bringing this out.

**Comment by JohnSully** [comment]
6.3 changed to pre-release on github to avoid others seeing issues in production.

**Comment by thenitai** [comment]
Yes, I just posted this on here as an FYI as it clearly can impact production (as it did with us). For the record, just flagging it here in Github doesn't help much if your users (like us) use package managers.
